src/models/CompanyProfile/CompanyProfile.ts

```diff
@@ -1,14 +1,17 @@
-import { AllowNull, Column, DataType, HasMany, Model, Table } from "sequelize-typescript";
+import { AllowNull, Column, DataType, HasMany, Model, Table, Is } from "sequelize-typescript";
 import { CompanyProfilePhoneNumber } from "../CompanyProfilePhoneNumber";
 import { CompanyProfilePhoto } from "../CompanyProfilePhoto";
+import { validateCuit, validateName } from "validations";
 
 @Table
 export default class CompanyProfile extends Model<CompanyProfile> {
   @AllowNull(false)
+  @Is("cuit", validateCuit)
```

Comment:
que hace este is? no lo encuentro porque es re dificil de googlear jajaja

Reply:
Aca explica que es: https://www.npmjs.com/package/sequelize-typescript

Reply:
Es un validator de sequelize-typescript